docs/e2e-test-guide.md

```diff
@@ -0,0 +1,164 @@
+# Argo CD Operator E2E Test Guide
+
+E2E tests are written using [KUTTL](https://kuttl.dev/docs/#install-kuttl-cli).
+
+## Requirements
+
+This test suite assumes that an Argo CD Operator is installed on the cluster or running locally using `make run`.
```

Comment:
I think it should be `make install run`, otherwise the CRDs won't be available on the cluster.

Reply:
Ack